src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -674,3 +674,29 @@ def create(self, vol_name: str, subvol_name: str, **kwargs):
             )
 
         return f'Subvolume {subvol_name} created successfully'
+
+
+@APIRouter('/cephfs/subvolume/group', Scope.CEPHFS)
+@APIDoc("Cephfs Subvolume Group Management API", "CephfsSubvolumeGroup")
+class CephFSSubvolumeGroups(RESTController):
+
+    def get(self, vol_name):
+        if not vol_name:
+            raise DashboardException(
+                'Error listing subvolume groups')
+        error_code, out, err = mgr.remote('volumes', '_cmd_fs_subvolumegroup_ls',
+                                          None, {'vol_name': vol_name})
+        if error_code != 0:
+            raise DashboardException(
+                'Error listing subvolume groups')
```

Comment:
nit: would be nice to have the vol name in conjunction with the message

Reply:
yup, that'd be good. Let's take it along with the [create subvolume group ](https://github.com/ceph/ceph/pull/52886/ )PR @Pegonzal . its hard to get a passing CI now.